render/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# This Dockerfile is based off the Google App Engine Python runtime image
+# https://github.com/GoogleCloudPlatform/python-runtime
+FROM gcr.io/google-appengine/python
+
+# Add metadata to Docker image
+LABEL maintainer="csse-education-research@canterbury.ac.nz"
+
+# Set terminal to be noninteractive
+ARG DEBIAN_FRONTEND=noninteractive
+ARG PROCESS_PORT=8080
+ARG FLASK_PRODUCTION=1
+
+ENV PORT ${PROCESS_PORT}
+ENV FLASK_PRODUCTION=${FLASK_PRODUCTION}
```

Comment:
This is to make it easy to change the port in future for dev and prod builds. The `ARG` command gets an argument or uses the default value and the `ENV` command is being used so I can access the environment variable within the container for the flask server.